packages/shared-ui/src/components/Header.tsx

```diff
@@ -211,13 +213,13 @@ export function HeaderContent({ children }: { children: React.ReactNode }) {
   return ReactDOM.createPortal(children, rootElement)
 }
 
-export function Header() {
+export function Header(props: HeaderProps) {
```

Comment:
For now, let's only have the images that we need for the `Header` component in the shared ui, stuff that is not specific to the Bridge UI, so:

```
- HeaderArbitrumLogoMainnet.png
- HeaderArbitrumLogoRinkeby.png
- HeaderArbitrumLogoGoerli.png
```

And the two icons for Discord and Twitter.

...

The other images you've moved are specific to the Bridge UI, and (at least for now) won't be used in any other projects.